### ISSUE TITLE
Create a `Builder` type for the `vulkano::sampler::Sampler` type.

### ISSUE THREAD

**Issue by mitchmindtree** [nannou-org/nannou#225]
The `Sampler` constructor is very verbose and allows for providing many parameters for which defaults could be easily provided:
```rust
pub fn new(
    device: Arc<Device>, 
    mag_filter: Filter, 
    min_filter: Filter, 
    mipmap_mode: MipmapMode, 
    address_u: SamplerAddressMode, 
    address_v: SamplerAddressMode, 
    address_w: SamplerAddressMode, 
    mip_lod_bias: f32, 
    max_anisotropy: f32, 
    min_lod: f32, 
    max_lod: f32
) -> Result<Arc<Sampler>, SamplerCreationError>
```

As the `vk` image examples currently demonstrate, they can be a PITA to use directly.

**Comment by mitchmindtree** [comment]
It could be worth proposing this upstream as the vulkano crate already makes heavy use of the builder pattern for its framebuffer, commandbuffer and graphics pipeline APIs.

**Comment by mitchmindtree** [comment]
Closed by #273.
